Generate/src/Medication-9-0-7/XIS-Server/Agreement/mp9-ma-fhir3-0-2-xis-1-1.xml

```diff
@@ -10,19 +10,16 @@
       <name value="Scenario 1.1"/>
       <description value="Scenario 1.1 - Medicatieafspraak gebruiksperiode ingangsdatum en einddatum"/>
       <nts:include value="xis-ma-vv-operation-search">
-         <nts:with-parameter name="description"
-                       value="Test XIS server to serve MedicationAgreement (NL: MedicatieAfspraak) - Medicatieafspraak gebruiksperiode ingangsdatum en einddatum"/>
+         <nts:with-parameter name="description" value="Test XIS server to serve MedicationAgreement (NL: MedicatieAfspraak) - Medicatieafspraak gebruiksperiode ingangsdatum en einddatum"/>
          <nts:with-parameter name="accept" value="xml"/>
-         <nts:with-parameter name="params"
-                       value="?category=http://snomed.info/sct|16076005&amp;medication.code=urn:oid:2.16.840.1.113883.2.4.4.10|6947&amp;_include=MedicationRequest:medication"/>
+         <nts:with-parameter name="params" value="?category=http://snomed.info/sct|16076005&amp;_include=MedicationRequest:medication"/>
       </nts:include>
       <nts:include value="assert-responseSearchBundleSuccess" scope="common"/>
       <nts:include value="assert-responseBundleContent" scope="common"/>
-      <nts:include value="assert-returnCount" scope="project">
-         <nts:with-parameter name="description"
-                       value="Confirm that the returned searchset Bundle contains 1 MedicationAgreement."/>
-         <nts:with-parameter name="expression"
-                       value="Bundle.entry.where(resource.is(MedicationRequest)).count() = 1"/>
+      <nts:include value="assert-returnCount-at-least-one-ma.xml" scope="project"/>      
```

Comment:
Had je hier niet het volgende kunnen gebruiken:
`<nts:include value="assert-returnCount" scope="project">
    <nts:with-parameter name="description" value="Confirm that the returned searchset Bundle contains at least 1 MedicationAgreement."/>
    <nts:with-parameter name="expression" value="Bundle.entry.where(resource.is(MedicationRequest)).count() >= 1"/>
</nts:include>`

Het is niet per se beter dan de oplossing die je nu hebt, maar dan maak je wel al gebruik van de bestaande include. Ter overweging.

Reply:
Ja! Dat had zeker gekund, maar dan zat ik met veel tekst in al die testscripts. Nu kan ik de omschrijving van die asserts op één plek beheren. :)